projects/github.com/fukuchi/package.yml

```diff
@@ -24,7 +24,8 @@ build:
   script: |
     ./autogen.sh
     ./configure --disable-dependency-tracking --prefix={{prefix}} $ARGS
-    VERSION={{version}} make
+    '#define VERSION "{{version}}"' >> config.h
```

Comment:
you might want to tailor this more closely for safety's sake:
```sh
if test "{{ hw.platform}}.{{ hw.arch }}" = "darwin.aarch64"; then
  # Oddly, VERSION isn't set, on exactly one platform. Perhaps related to:
  # https://github.com/Qalculate/libqalculate/issues/2
  echo '#define VERSION "{{version}}"' >> config.h
fi
```

Reply:
Looks like it passed build, though (the failure is just the stupid 500 deno deps issue).